src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const basicDiscount = 20;
+  const extendedDiscount = 50;
+
+  const rentalCost = days * dayCost;
+
+  return days > 6
+    ? rentalCost - extendedDiscount
+    : days > 2
+      ? rentalCost - basicDiscount
+      : rentalCost;
```

Comment:
nested ternaries are not considered good practice
better replace with `if (...) { return ... }`